docs/extend/frontend.md

```diff
@@ -308,7 +308,7 @@ DiscussionPage.prototype.view = function() {
 }
 ```
 
-will turn Flarum's discussion pages into proclamations of "Hello World". How creative!
+...will turn Flarum's discussion pages into proclamations of "Hello World". How creative!
```

Comment:
Why this change?

Reply:
Because I spent a few minutes trying to figure out why there was a lower case letter starting a sentence, until I noticed it was continuing the sentence above the code block.